db/migrate/20210302124301_create_places.rb

```diff
@@ -1,8 +1,10 @@
 class CreatePlaces < ActiveRecord::Migration[6.1]
   def change
     create_table :places do |t|
-
+      t.string  :place_name
+      t.string  :address
```

Comment:
`place_name` と `address` は、NOT NULL制約をかけたほうが良いですね。
どちらも、 `Place` には必須項目だと思うので。

```suggestion
      t.string  :place_name, null: false
      t.string  :address, null: false
```